ca_on_ajax/people.py

```diff
@@ -1,49 +1,43 @@
-from utils import CanadianScraper, CanadianPerson as Person, CONTACT_DETAIL_TYPE_MAP
+from utils import CanadianScraper, CanadianPerson as Person
 
-import re
-
-COUNCIL_PAGE = 'http://www.ajax.ca/en/insidetownhall/mayorcouncillors.asp'
+COUNCIL_PAGE = 'https://www.ajax.ca/en/inside-townhall/council-members.aspx'
 
 
 class AjaxPersonScraper(CanadianScraper):
     def scrape(self):
         page = self.lxmlize(COUNCIL_PAGE)
 
-        councillors = page.xpath('//ul[@class="subNav top"]/li/ul//li/a[contains(text(), "Councillor")] | //ul[@class="subNav top"]/li/ul//li/a[contains(text(), "Mayor")]')
-        assert len(councillors), 'No councillors found'
+        councillors = page.xpath('//table[@class="councilTable"]')
         for councillor in councillors:
-            name = councillor.text_content()
+            image = councillor.xpath('.//@src')[0]
+            councillor_name = councillor.xpath('.//tr/td[1]/p[1]/img/@alt')
+            assert len(councillors), 'No councillors found'
 
-            url = councillor.attrib['href']
-            page = self.lxmlize(url)
+            for i in councillor_name:
```

Comment:
Why is this a for loop? Shouldn't we just add `[0]` at the end of the `councillor_name = …` line above?